### ISSUE TITLE
Problem in displaying a list of vectors

### ISSUE THREAD

**Issue by renkun-ken** [IRkernel/IRkernel#283]
Run the following code 

``` r
lapply(1:3, function(i) {
    x <- rnorm(1000)
    y <- i * x + rnorm(1000) / i
    m <- lm(y ~ x)
    coef(m)
})
```

and the output is displayed with some problem.

![jupyter_display_issues](https://cloud.githubusercontent.com/assets/4662568/13806129/8c9e953c-eb94-11e5-9634-a20c7e7f7fe7.png)

``` r
lapply(1:10, numeric)
```

and the display has some inconsistencies:
![jupyter_display_list_of_integers](https://cloud.githubusercontent.com/assets/4662568/13806159/addb71c0-eb94-11e5-90d8-69a611e9cc9b.png)

**Comment by flying-sheep** [comment]
the first one is a problem, yes.

the second one comes from two things:

1. we rely on bootstrap classes (in this case `.list-inline`) and the jupyter notebook throws together all kinds of style, resulting in [its `padding` and `margin` for `.rendered_html ol`](https://github.com/jupyter/notebook/blob/master/notebook/static/notebook/less/renderedhtml.less) overriding bootstrap’s for `.list-inline`
2. single numbers are rendered as is, not in a `ol.list-inline`, which means the margin/padding is not buggy here.

**Comment by flying-sheep** [comment]
fixed the second one in jupyter/notebook#1220

**Comment by flying-sheep** [comment]
the first one is really weird.

apparently if there’s floating stuff inside of a `<li/>`, the list numbering gets pushed up. can’t see a way to fix it.
